script/cibuild

```diff
@@ -2,15 +2,50 @@
 
 set -e
 
-# setup environment
+# GC customizations
+export RUBY_GC_MALLOC_LIMIT=79000000
+export RUBY_GC_HEAP_INIT_SLOTS=800000
+export RUBY_HEAP_FREE_MIN=100000
+export RUBY_HEAP_SLOTS_INCREMENT=400000
+export RUBY_HEAP_SLOTS_GROWTH_FACTOR=1
+
 export PATH="/usr/share/rbenv/shims:$PATH"
+export RACK_ROOT=$(cd "$(dirname $0)"/.. && pwd)
+export RACK_ENV="test"
+export RAILS_ENV="test"
+export RBENV_VERSION="2.2.3"
```

Comment:
@brianamarie @stoe is there a particular reason we're testing against this version? I get the following when I try to test with this version: 

```sh
hostname: Hector's
pwd: /Users/halfaro1/github/training-kit
git version 2.11.0

==> Installing gem dependencies…
rbenv: bundle: command not found

The `bundle' command exists in these Ruby versions:
  2.0.0-github6
  2.3.0
  2.3.1

rbenv: bundle: command not found

The `bundle' command exists in these Ruby versions:
  2.0.0-github6
  2.3.0
  2.3.1
```